.github/workflows/lint_actions.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install requirements
         run: |
           set -eux
-          go install github.com/rhysd/actionlint/cmd/actionlint@7040327ca40aefd92888871131adc30c7d9c1b6d
+          go install github.com/rhysd/actionlint/cmd/actionlint@97ad72abf96ba516f4a49f2da567165eaebf7e5a
```

Comment:
`go install github.com/rhysd/actionlint/cmd/actionlint@v1.6.15` also works, and it might be bit clearer on which versions we use. Disclaimer: this is a TIL for me, I don't know go :D

Reply:
Good point. I can do that as part of a follow up PR>